Apps/Sandcastle/gallery/Atmosphere Color.html

```diff
@@ -63,7 +63,9 @@
 function startup(Cesium) {
     'use strict';
 //Sandcastle_Begin
-var viewer = new Cesium.Viewer('cesiumContainer');
+var viewer = new Cesium.Viewer('cesiumContainer',{
+sceneModePicker:false
+});
```

Comment:
Just to keep with the general style of the Cesium code and examples, please indent the line `sceneModePicker: false` (4 spaces). We also generally add a space between function arguments, so `'cesiumContainer',{` -> `'cesiumContainer', {`.